### PR TITLE
Modify description of automated benchmarking talk

### PR DESCRIPTION
**Pull request by sipian**
@krasi-georgiev

Krasi and I decided to give this talk together. :smiley:  
Also modified the talk description

### CODE REVIEW

File: content/2018-munich/speakers/harsh-agarwal.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Harsh Agarwal
+---
+
+## Harsh Agarwal
+
+I am an undergraduate student from IIT Hyderabad and a developer from India.
+
+I was introduced to Prometheus when I built my college's monitoring infrastructure on top of Prometheus.
+
+This motivated me to contribute to Prometheus and am now a Google Summer of Code 2018 intern working on Prometheus' benchmarking and e2e testing tool.
```
Comment (juliusv) [inline content/2018-munich/speakers/harsh-agarwal.md]: "and am" -> "and I am"

File: content/2018-munich/talks/automated-prometheus-benchmarking.md [hunk 2]
```diff
@@ -4,8 +4,13 @@ title: Automated Prometheus Benchmarking - Pushing It to Its Limits until It Bre
...
 
-Speaker: [Krasi Georgiev](/2018-munich/speakers/krasi-georgiev/)
+Speakers:
 
-I want to present the new automated benchmarking and scalability tests that we should have by the time of PromCon.
+* [Krasi Georgiev](/2018-munich/speakers/krasi-georgiev/)
+* [Harsh Agarwal](/2018-munich/speakers/harsh-agarwal/)
 
-I will document why we need them and the most interesting milestones of building the CI. Will briefly mention the decisions and directions that lead us to the final result.
+We will show you how we are trying to make Prometheus more stable by running automated benchmarking for risky PRs and before every release. In other words lets try to break it in our tests so it doesn't break in your production.
```
Comment (juliusv) [inline content/2018-munich/talks/automated-prometheus-benchmarking.md]: Comma after "words"

"lets" -> "let's"

**Comment by sipian** [comment]
@juliusv 
Thanks for pointing out the errors.

**Comment by juliusv** [comment]
:+1: @RichiH is this fine?

**Comment by RichiH** [comment]
@sipian I talked to Krasi about this as well; please try and make sure to test how you do handover. Multi-speaker talks tend to have hiccups around any speaker switch, which tends to make the audience lose track of the content for some time.

**Comment by krasi-georgiev** [comment]
@RichiH thanks, I think it will go well as I have a good idea how to separate it. Similar to what they do at Docker con sometime. One person talks about changes and roadmap and someone else does a short demo and this is what we will do as well. will do everything from a single laptop

**Pull request merged**